eta/core/models.py

```diff
@@ -1472,70 +1496,103 @@ class ModelsManifest(Serializable):
 
     _MODEL_CLS = Model
 
-    def __init__(self, models=None):
+    def __init__(self, models=None, name=None, url=None):
         """Creates a ModelsManifest instance.
 
         Args:
             models: a list of Model instances
+            name (optional): a name for the manifest
+            url (optional): the source location of the manifest
         """
-        self.models = models or []
+        if models is None:
+            models = []
+
+        if name is not None:
+            subdir = os.path.join(*name.split("/"))
```

Comment:
This seems like it is a no-op? You're splitting then joining it back together again. Do you mean to take out the file name? If so, why not use `os.path.dirname(name)`?

Reply:
It's for windows. Converting `org-name/model-name` to `org-name\model-name` if necessary

Reply:
mm I see. A little confusing that `name` is assigned to `subdir` then.